posthog/__init__.py

```diff
@@ -214,18 +214,25 @@ def alias(
     The same concept applies for when a user logs in.
 
     An `alias` call requires
-    - `previous distinct id` the unique ID of the user before
-    - `distinct id` the current unique id
+    - `distinct id` the current unique id of the user (normally the id in your database)
+    - `alias distinct id` the alias id you want to attach to the user, such as the anonymous session id or another ID like their email
+
 
     For example:
     ```python
-    posthog.alias('anonymous session id', 'distinct id')
+    posthog.alias('distinct id', 'anonymous session id')
+    ```
+
+    or
+
+    ```python
+    posthog.alias('distinct id', 'users-email@posthog.com')
```

Comment:
this looks like a bad example. The `users-email@posthog.com` doesn't look like an anonymous session ID to me